.gitignore

```diff
@@ -2,7 +2,4 @@ node_modules
 dist
 coverage
 reports
-
-# IntelliJ Idea
-.idea/
-*.iml
+settings.json
```

Comment:
We should keep the IntelliJ Idea ignores, since we have a developer that uses that editor.

Reply:
I re-added the intellij stuff